data/yam/agama/auto/sap_default_no_scc.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
why in the names of all the files in the PR we are loosing `_no_scc.json` ?

Reply:
These no_scc profiles renamed and missed, I added them.